pkg/cmd/pulumi/stack/secrets.go

```diff
@@ -107,7 +108,7 @@ func createSecretsManagerForNewStack(
 	if err != nil {
 		ps = &workspace.ProjectStack{}
 	} else {
-		ps, err = loadProjectStackByReference(project, stackRef)
+		ps, err = loadProjectStackByReference(project, stackRef, nil)
```

Comment:
```suggestion
		ps, err = loadProjectStackByReference(project, stackRef, nil /* sink */)
```
Would be helpful in the other places as well.